en/docs/install-and-setup/setup/sso/configuring-sso-for-admin-portal-using-saml.md

```diff
@@ -0,0 +1,230 @@
+
+        <table>
+        <tbody>
+            <tr>
+                <td><b>Issuer</b></td>
+                <td>Specify the Issuer</td>
+            </tr>
+            <tr>
+                <td><b>Assertion Consumer URLs</b></td>
+                <td><code>https://{apim-ip}:9443/admin/jagg/jaggery_acs.jag</code></td>
+            </tr>
+            <tr>
+                <td><b>Response Signing Algorithm</b></td>
+                <td><code>http://www.w3.org/2001/04/xmldsig-more#rsa-sha256</code></td>
+            </tr>
+            <tr>
+                <td><b>Enable Response Signing</b></td>
+                <td>true</td>
+            </tr>
+            <tr>
+                <td><b>Enable Signature Validation in Authentication Requests and Logout Requests</b></td>
+                <td>true</td>
+            </tr>
+            <tr>
+                <td><b>Enable Single Logout</b></td>
+                <td>true</td>
+            </tr>
+            <tr>
+                <td><b>Enable Attribute Profile</b></td>
+                <td>true</td>
+            </tr>
+            <tr>
+                <td><b>Include Attributes in the Response Always</b></td>
+                <td>true</td>
+            </tr>
+        </tbody>
+        </table>
+
+        The following image shows the sample values for SAML2 Web SSO Configuration:
+
+        [![SAML configuration in Service Provider]({{base_path}}/assets/img/setup-and-install/saml-configuration-in-service-provider.png)]({{base_path}}/assets/img/setup-and-install/saml-configuration-in-service-provider.png)
+
+        !!! Info "In multi-tenanted environments"
+            Carry out the instruction given below for all the tenants to be able to sign in to the Admin Portal in a multi-tenanted environment.
+
+            1. Click the **SaaS Application** option that appears after registering the service provider.
+
+            [![SaaS configuration in service provider]({{base_path}}/assets/img/setup-and-install/saas.png)]({{base_path}}/assets/img/setup-and-install/saas.png)            
+
+            If you do not select the **SaaS Application** option, only users in the current tenant domain will be allowed to sign in to the portal. You will need to register separate service providers for portals from each tenant.
+
+    2.  Register the Service Provider.
+
+        ??? Info "Enable a tenant-specific SSO for the Publisher and Developer Portal"
+
+            To enable a tenant-specific SSO with IS 5.10.0, enable the **Use tenant domain in local subject identifier** option under the **Local & Outbound Authentication Configuration** section.
+
+            [![Enable tenant domain in local sub identifier]({{base_path}}/assets/img/setup-and-install/enable-tenant-domain-in-local-sub-identifier.png)]({{base_path}}/assets/img/setup-and-install/enable-tenant-domain-in-local-sub-identifier.png)
+
+4.  Upload the public certificate of the API Manager by selecting **Select SP Certificate Type**.
+
```

Comment:
Images and files can not have uppercase characters in it.
This image will not render due to the upper case characters in upload-certificate-in-SP-for-saml2-sso.png.

Reply:
As the author is unavailable for the next couple of days, [1] has been created to expedite the fix.

[1] https://github.com/wso2/docs-apim/issues/2142